pkgs/applications/editors/quartus-prime/default.nix

```diff
@@ -110,10 +138,40 @@ in buildFHSUserEnv {
     xorg.libXrender
   ];
 
-  extraInstallCommands = ''
+  passthru = {
+    inherit quartus;
+  };
+
+  extraInstallCommands = let
+    quartusExecutables = (map (c: "quartus/bin/quartus_${c}") [
+      "asm" "cdb" "cpf" "drc" "eda" "fit" "jbcc" "jli" "map" "pgm" "pow"
+      "sh" "si" "sim" "sta" "stp" "tan"
+    ]) ++ [ "quartus/bin/quartus" ];
+
+    qsysExecutables = (map (c: "quartus/sopc_builder/bin/qsys-${c}") [
+      "generate" "edit" "script"
+    ]);
```

Comment:
Just out of curiosity, why not merge this list to `quartusExecutables`? It will ease on you in the shell script following this line.

Reply:
No particular reason, I had two separate loops in some previous revisions so this is left over from that. That does remind me I wanted to add the ModelSim binary as well!